src/YoloObjectDetector.cpp

```diff
@@ -379,7 +362,11 @@ void YoloObjectDetector::yolo()
     detect_thread = std::thread(&YoloObjectDetector::detectInThread, this);
     
     generate_image_cp(buff_[(buffIndex_ + 1)%3], disp_);
-    rtsp_streamer_.publish_rtsp_stream(disp_);
+    if (writer_configured_)
```

Comment:
is there something we would like to do if the writer is not configured?
Also, can we remove the commented lines from this file?

Reply:
If the writer is not configured we simply discard the current frame, until we manage to configure the writer and send the buffer through. 

The commented lines form part of the original code before some slight modifications we did on top. I would not remove them for now as its a 3rd party repo mixed with our changes.